profiles/models.py

```diff
@@ -0,0 +1,39 @@
+from operator import mod
+from django.db import models
+# Create your models here.
+
+class User(models.Model):
```

Comment:
this is going to be replaced by some other model that represents mentor/mentee. pls ignore. also not referenced in any other part of the code.